src/java/htsjdk/tribble/TribbleIndexedFeatureReader.java

```diff
@@ -271,6 +273,27 @@ private void readHeader() throws IOException {
         return new WFIterator();
     }
 
+    //Visible for testing
+    static boolean isGZIPPath(final String path) {
```

Comment:
There are similar functions in IOUtil, but they accept another extension for gzip, and its unclear to me what to what extent these should overlap or not. So I'm leaving this here as a narrow extension to the local test that was previously being done.